sassanh/qnvim
# Make plugin compatible with QtCreator 5.0.2

This changed API a little bit. 

## Test Plan

1. Make a **clean** build[^1] and run QtCreator from `%{sourceDir}\external\qtcreator\dist-Windows\bin\qtcreator.exe` and args `-pluginpath %{buildDir}\lib\qtcreator\plugins`. This should load the plugin.
2. Check, that it still works (in our case saving is concerned).


[^1]: This is to prevent Qt Creator from loading an old version of the plugin.
